sdc/tests/tests_perf/test_perf_utils.py

```diff
@@ -272,35 +337,18 @@ def dump(self):
         """
         Dump performance testing results from global data storage to excel
         """
-        # openpyxl need to be installed
-
-        try:
-            with pandas.ExcelWriter(self.perf_results_xlsx) as writer:
-                self.grouped_data.to_excel(writer)
-        except ModuleNotFoundError as e:
-            msg = 'Could not dump the results to "%s": %s'
-            self.logger.warning(msg, self.perf_results_xlsx, e)
-
-        try:
-            with pandas.ExcelWriter(self.raw_perf_results_xlsx) as writer:
-                self.test_results_data.to_excel(writer, index=False)
-        except ModuleNotFoundError as e:
-            msg = 'Could not dump raw results to "%s": %s'
-            self.logger.warning(msg, self.raw_perf_results_xlsx, e)
+        for d in self.drivers:
+            d.dump_grouped_data(self.grouped_data, self.logger)
+            d.dump_test_results_data(self.test_results_data, self.logger)
 
     def load(self):
         """
         Load existing performance testing results from excel to global data storage
         """
-        raw_perf_results_xlsx = Path(self.raw_perf_results_xlsx)
-        if raw_perf_results_xlsx.exists():
-            with raw_perf_results_xlsx.open('rb') as fd:
-                # xlrd need to be installed
-                try:
-                    self.test_results_data = pandas.read_excel(fd)
-                except ModuleNotFoundError as e:
-                    msg = 'Could not load previous results from %s: %s'
-                    self.logger.warning(msg, raw_perf_results_xlsx, e)
+        if self.default_driver:
+            test_results_data = self.default_driver.load(self.logger)
+            if test_results_data is not None:
+                self.test_results_data = test_results_data
```

Comment:
I think we can try to load data via each driver from the all specified drivers until we get the data something like:
```
for d in self.drivers:
    test_results_data = d.load()
    if test_results_data:
        self.test_results_data = test_results_data
        break
```

Reply:
ok